daisy/compute/compute.go

```diff
@@ -848,6 +860,17 @@ func (c *client) InstanceStopped(project, zone, name string) (bool, error) {
 	}
 }
 
+// ResizeDisk resizes a GCE persistent disk. You can only increase the size of the disk.
+func (c *client) ResizeDisk(project, zone string, d *compute.Disk) error {
```

Comment:
Take name of disk and compute.DisksResizeRequest

func (c *client) ResizeDisk(project, zone, disk string, drr *compute.DisksResizeRequest) error {